groups/README.md

```diff
@@ -0,0 +1,28 @@
+Hi there! Welcome to Kevala! 
+
+As a member of the engineering team, you've been directed towards this repo to set up your Google Cloud console access. 
+
+To get started, go ahead and clone this repository and checkout a new branch
+
+Open main.tf. Inside you'll find multiple `module` declarations like the block below. 
+
+* If you're creating a new group, go ahead and copy-paste another group's declaration - don't forget to change the friendly name of the module (the "example_kevala_group" bit)! If you run into issues, Infra team is here to help! 
```

Comment:
This tutorial text needs dedicated wordsmithing time when it comes time to handing this over to users, but leaving as-is for now